README.md

```diff
@@ -146,6 +147,15 @@ didReceiveDataChunk:(NSData *)data
 ```
 Be sure to render YES in your delegate to tell the data loader that you support chunks, and to set the requests chunks property to YES.
 
+### Rate limiting specific endpoints
+If you specify a rate limiter in your service, you can give it a default requests per second metric which it applies to all requests coming out your app. (See "Creating the SPTDataLoaderService"). However, you can also specify rate limits for specific HTTP endpoints, which may be useful if you want to forcefully control the rate at which clients can make requests to a backend that does large amounts of work.
```

Comment:
How about linking references up?

```md
(See “[Creating the `SPTDataLoaderService`](#creating-the-sptdataloaderservice)”
```

That change also changes to use [correct quote characters](http://smartquotesforsmartpeople.com/).